build failed

Error output from Xcode build:
↳
    ** BUILD FAILED **


Xcode's output:
↳
    /Users/apple/git_flutter/flutter/.pub-cache/hosted/pub.flutter-io.cn/workmanager-0.1.0/ios/Classes/SwiftWorkmanagerPlugin.swift:98:156: error: argument type 'FlutterEngine' does not conform to expected type 'FlutterBinaryMessenger'
            var backgroundMethodChannel: FlutterMethodChannel? = FlutterMethodChannel(name: BackgroundMethodChannel.channelName, binaryMessenger: flutterEngine!)
                                                                                                                                                  ~~~~~~~~~~~~~^
                                                                                                                                                                 as! FlutterBinaryMessenger

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,3 +1,8 @@
+# 0.1.1
+
+* iOS:
+  * Added support for Flutter 1.9.1
+
 # 0.1.0
 * First API stable release
 * Android: 

--- a/README.md
+++ b/README.md
@@ -11,7 +11,7 @@ This is especially useful to run periodic tasks, such as fetching remote data on
 
 ```yaml
 dependencies:
-  workmanager: ^0.1.0
+  workmanager: ^0.1.1
 ```
 ```shell script
 flutter pub get

--- a/ios/Classes/SwiftWorkmanagerPlugin.swift
+++ b/ios/Classes/SwiftWorkmanagerPlugin.swift
@@ -95,7 +95,7 @@ extension SwiftWorkmanagerPlugin {
         flutterEngine!.run(withEntrypoint: flutterCallbackInformation.callbackName, libraryURI: flutterCallbackInformation.callbackLibraryPath)
         SwiftWorkmanagerPlugin.flutterPluginRegistrantCallback?(flutterEngine!)
         
-        var backgroundMethodChannel: FlutterMethodChannel? = FlutterMethodChannel(name: BackgroundMethodChannel.channelName, binaryMessenger: flutterEngine!)
+        var backgroundMethodChannel: FlutterMethodChannel? = FlutterMethodChannel(name: BackgroundMethodChannel.channelName, binaryMessenger: flutterEngine!.binaryMessenger)
         
         func cleanupFlutterResources() {
             flutterEngine?.destroyContext()

--- a/pubspec.yaml
+++ b/pubspec.yaml
@@ -1,6 +1,6 @@
 name: workmanager
 description: Flutter Workmanager. This plugin allows you to schedule background work on Android and iOS.
-version: 0.1.0
+version: 0.1.1
 authors: 
   - Tim Rijckaert <tim.rijckaert@gmail.com>
   - Jeremie Vincke <jeremie@movify.com>
@@ -10,7 +10,7 @@ repository: https://github.com/vrtdev/flutter_workmanager
 issue_tracker: https://github.com/vrtdev/flutter_workmanager/issues
 
 environment:
-  sdk: ">=2.2.0 <3.0.0"
+  sdk: ">=2.5.0 <3.0.0"
 
 dependencies:
   flutter:

